### PR TITLE
Add endpoint for minsky post by subreddit recommendations

### PR DESCRIPTION
**Pull request by keithblaha**
:eyeglasses: @scarow @schwers

### CODE REVIEW

File: src/apis/SubredditsByPost.es6.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { isEmpty } from 'lodash/lang';
```
Comment (schwers) [inline src/apis/SubredditsByPost.es6.js]: 🔕  In general the best practice for importing this is:

```javascript
import isEmpty from 'lodash/isEmpty';
```

We should keep it as is for consistency and take a cleanup pass soon.

**Comment by schwers** [comment]
:fish:

**Comment by keithblaha** [comment]
:haircut_woman: sorry to modify already :fish:ed PR but would like to piggy back this additional endpoint addition and fix an issue

**Comment by scarow** [comment]
Should the file names here change too? `SubredditsByLink` instead of `post`?

File: src/index.es6.js [hunk 2]
```diff
@@ -30,6 +30,8 @@ import SearchEndpoint from './apis/SearchEndpoint';
 import SubredditAutocomplete from './apis/SubredditAutocomplete';
 import subscriptions from './apis/subscriptions';
 import SubredditEndpoint from './apis/SubredditEndpoint';
+import SubredditsByPost from './apis/SubredditsByPost';
+import SubredditsToPostsByPost from './apis/SubredditsToPostsByPost';
```
Comment (scarow) [inline src/index.es6.js]: Maybe change the names here too -- keep it consistent across every app

**Comment by scarow** [comment]
:haircut_man:

**Comment by keithblaha** [comment]
So I had thought about the naming stuff at first and its problematic because within r2 they're called links, but consistently throughout 2X they're called posts. I decided it was least confusing to just stick with local naming across the board, even if that's slightly confusing across the platforms.

**Comment by scarow** [comment]
O0o0o that is annoying. Very well.

**Comment by scarow** [comment]
lgtm 🐟

**Comment by keithblaha** [comment]
added another endpoint... sorry to mess with it given it was already fished, but :haircut_woman:

**Comment by keithblaha** [comment]
bump - this was modified after the last fish (apologies again for that)

**Comment by keithblaha** [comment]
:eyeglasses: @prashtx @nramadas

**Pull request merged**